uploader.v2.json

```diff
@@ -3368,7 +3354,22 @@
             "glob": "/sys/class/scsi_host/host[0-9]*/eh_deadline",
             "symbolic_name": "scsi_eh_deadline",
             "pattern": []
-        }
+        },
+	{
+            "glob": "/var/log/ceph/()ceph.log*",
```

Comment:
Please change this entry to `"glob": "/var/log/ceph/ceph.log*",`.

Reply:
Done